libsoftwarecontainer/src/gateway/cgroups/cgroupsgateway.cpp

```diff
@@ -30,32 +30,32 @@ CgroupsGateway::CgroupsGateway()
 {
 }
 
-ReturnCode CgroupsGateway::readConfigElement(const json_t *element)
+bool CgroupsGateway::readConfigElement(const json_t *element)
 {
     try {
         m_parser.parseCGroupsGatewayConfiguration(element);
     } catch (CgroupsGatewayError &e) {
         log_error() << "Could not parse CGroups configuration element";
-        return ReturnCode::FAILURE;
+        return false;
     }
 
-    return ReturnCode::SUCCESS;
+    return true;
 }
 
 bool CgroupsGateway::activateGateway()
 {
-    ReturnCode success = ReturnCode::FAILURE;
+    bool success = false;
     auto cgroupSettings = m_parser.getSettings();
     for (auto& setting: cgroupSettings) {
-        success = getContainer()->setCgroupItem(setting.first, setting.second);
-        if (success != ReturnCode::SUCCESS) {
+        success = isSuccess(getContainer()->setCgroupItem(setting.first, setting.second));
```

Comment:
I might be misreading this since it's a big diff in total, but this looks like we're adding a line that uses "isSuccess". Is this some other isSuccess than the ReturnCode related one?